src/main/java/use_case/show/ShowInteractor.java

```diff
@@ -58,8 +61,14 @@ public void execute(ShowInputData showInputData) throws JsonProcessingException
                     if (dailyData != null) {
                         String closingPrice = dailyData.get("4. close");
                         Double price = Double.valueOf(closingPrice);
-                        dateToNetWorth.put(dateStringWithoutTime,
-                                dateToNetWorth.getOrDefault(dateStringWithoutTime, 0.0) + stock.getQuantity() * price);
+                        if(stock instanceof Short){
+                            dateToNetWorth.put(dateStringWithoutTime, dateToNetWorth.getOrDefault(dateStringWithoutTime,
+                                    0.0) + stock.getQuantity() * (initialPrice - price));
```

Comment:
Since we are calculating the networth of the portfolio if the value of the short goes up then the users networth goes down thus we need to subtract the original price